src/patterns/InPlaceFilters/FilterDropdown.tsx

```diff
@@ -0,0 +1,132 @@
+import classnames from 'classnames';
+import React from 'react';
+import Button from '../../components/Button/Button';
+import { ButtonVariants } from '../../components/Button/types';
+import FilterHeader from '../../components/Filter/FilterHeader';
+import FilterInput from '../../components/Filter/FilterInput';
+import Text from '../../components/Text/Text';
+import { TextVariants } from '../../components/Text/types';
+import { px } from '../../utils';
+import { FilterDimension, FilterDropdownProps } from './types';
+import { getFilterDimensions } from './utils';
+
+export const FilterDropdown = React.forwardRef<HTMLDivElement, FilterDropdownProps>(
```

Comment:
can we rename this `FilterDropdownMenu` I got confused trying to understand it.

Reply:
I'm a fan of putting the component typescript props definition inside the component file itself, could you move the `FilterDropdownProps` definition here and rename to `FilterDropdownMenuProps`

Reply:
I think it might make sense to make a `FilterDropdownMenuDesktop` vs `FilterDropdownMenuMobile` and render one or the other from the parent component level with the `SSRMediaQuery`.  Take a look at how we did it here with `MobileLanguageSelector` https://github.com/PhillipsAuctionHouse/seldon/blob/eee503aaa50fc80b7155740052da1e08b1a23c43/src/patterns/LanguageSelector/LanguageSelector.tsx#L23